solr/core/src/java/org/apache/solr/servlet/CoordinatorHttpSolrCall.java

```diff
@@ -314,6 +182,19 @@ public CloudDescriptor getCloudDescriptor() {
     };
   }
 
+  /**
+   * Overrides the MDC context as the core set was synthetic core, which does not reflect the
+   * collection being operated on
+   */
+  private static void setMDCLoggingContext(String collectionName) {
```

Comment:
We use Google Java code standards, thus use "Mdc" capitalization.  Plenty of old code did a mis-mash of things, MDCLoggingContext being one.